share/doc/xml/acis_bkgrnd_lookup.xml

```diff
@@ -26,19 +26,19 @@
 
       <PARA>
         - <HREF
-	link="http://cxc.harvard.edu/cal/Acis/Cal_prods/bkgrnd/current/background.html">ACIS
+	link="https://cxc.harvard.edu/cal/Acis/Cal_prods/bkgrnd/current/background.html">ACIS
```

Comment:
The url now has to include the 'cfa' or users will get an "incorrectly configured certificate" or some such warning when they visit the page.

Reply:
I don't see this behavior on firefox or chrome - i.e. going to https://cxc.harvard.edu/cal/Acis/Cal_prods/bkgrnd/current/background.html (r any other page I've tried) doesn't produce a warning for me.